lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +70,36 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding="utf-8") as amur:
+            config_dto = json.load(amur)
+        return ConfigDTO(**config_dto)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        if (not isinstance(self.extract.seed_urls, list) or
+                not all(re.match(r"https?://(www.)?portamur\.ru/news", url) for url in self.extract.seed_urls)):
+            raise IncorrectSeedURLError
+
+        if not isinstance(self.extract.total_articles, int) or self.extract.total_articles <= 0:
+            raise IncorrectNumberOfArticlesError
+
+        if not 1 <= self.extract.total_articles <= 150:
```

Comment:
I believe you check the left condition twice (see line 85)

Reply:
Measure twice, cut once